roles/ara_api/tasks/pre-requirements.yaml

```diff
@@ -27,12 +27,13 @@
 
 # Only attempt to elevate privileges if there are any missing packages
 - when: ara_api_missing_packages | length > 0
-  become: yes
+  become: true
   block:
     # The apt module can not be used for this since it installs python-apt
```

Comment:
I haven't had issues with the installation of python-apt so far, let's see what the CI makes of this :-)